### PR TITLE
Create torchboard integration workflow

### PR DESCRIPTION
**Pull request by rusito-23**
* Create configuration file to allow package installation and testing
* Setup placeholder tests
* Create isort configuration file to match CI and local checks
* Setup CI steps:
    - prepare environment using py 3.8.0
    - install dependencies
    - lint all files with black (does not modify)
    - lint all imports with isort (does not modify)
    - run all tests
    - gather and upload code coverage to codecov.io
* Create scripts to locally test/coverage and verify package can be installed
* Add two nice badges to the README to show coverage & actions status

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[metadata]
+name = torchboard
+version = 0.1.0
+description = "Clumsy TensorBoard copycat for PyTorch"
+author = "Igor Andruskiewitsch, Marco Martinelli"
+author_email = "i.andruskiewitsch23@gmail.com, marcocba@gmail.com"
```
Comment (rusito-23) [inline setup.cfg:6]: @MarcoCBA to confirm e-mail

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[metadata]
+name = torchboard
+version = 0.1.0
+description = "Clumsy TensorBoard copycat for PyTorch"
+author = "Igor Andruskiewitsch", "Marco Martinelli"
+author_email = "i.andruskiewitsch23@gmail.com", "marcocba@gmail.com"
```
Comment (rusito-23) [inline setup.cfg]: @MarcoCBA to confirm e-mail

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+license = "MIT"
+readme = "README.md"
+python = "^3.8"
+repository = "https://github.com/TorchBoard/TorchBoard"
+
+[options]
+packages = find:
```
Comment (rusito-23) [inline setup.cfg:13]: FIXME: This is installing both `torchboard` and `tests` as packages, we should specify that the source folder is `torchboard` only!

File: setup.cfg (hunk 2 above)
Comment (smartinellimarco) [reply setup.cfg]: smartinellimarco@gmail.com

**Review by smartinellimarco — APPROVED** [review]
🙈

**Pull request merged**